generators/heroku/index.js

```diff
@@ -293,9 +324,12 @@ module.exports = class extends BaseGenerator {
                 const done = this.async();
 
                 const regionParams = this.herokuRegion !== 'us' ? ` --region ${this.herokuRegion}` : '';
-
+                let buildPackParams = ' --buildpack heroku/jvm';
+                if (this.buildTool === 'gradle') {
```

Comment:
By default heroku is doing auto selection. It results in two buildpacks: node + gradle or jvm. This results in the frontend build executed twice. One time by the node buildpack and a second time by the java build.

Reply:
EDIT: it is not auto detection we set it explicitly. Think that is not needed. Therefore we don't set anything for jar deployments and heroku rejects deploying a jar then. Will further investigate.